src/data/tableColumns/tableWithTabs.js

```diff
@@ -0,0 +1,65 @@
+export default [
+    {
+        "name": "id",
+        "label": "id",
+        "field": "id",
+        "align": "left",
+        "sortable": true
+    },
+    {
+        "name": "name",
+        "label": "name",
+        "field": "name",
```

Comment:
Remove the need to repeat these?

All we need to define should just be either name, where field/label would be fallback of it

Reply:
In utils created `getTableColumns` function to handle other attributes.